cli/tests/test_squash_command.rs

```diff
@@ -690,6 +691,14 @@ fn test_squash_from_multiple() {
     insta::assert_snapshot!(stdout, @r###"
     f
     "###);
+
+    // Empty squash shouldn't crash (could be noop)
```

Comment:
Should we output "Nothing changed." in that case like some other commands do?

Reply:
Nvm, since a new wc is created

Reply:
It could be changed to noop (with "Nothing changed." message), but I'm not sure if we want to add a special case for this unusual situation. So I just added a test capturing the current behavior.